RecoEgamma/EgammaTools/src/EnergyScaleCorrection.cc

```diff
@@ -359,7 +361,21 @@ EnergyScaleCorrection::CorrectionCategory::CorrectionCategory(const std::string&
     r9Min_ = -1;
     r9Max_ = 0.94;
   };	
-  
+  // R9 region
+  p1 = category.find("-R9");
+  p2 = p1 + 1;
+  if(p1 != std::string::npos) {
+    p1 = category.find("_", p1);
+    p2 = category.find("_", p1 + 1);
+    r9Min_ = TString(category.substr(p1 + 1, p2 - p1 - 1)).Atof();
```

Comment:
why is TString needed here?
std::stof apparently works in all other parts of this code.